server/test/repository/ApplicationRepositoryTest.java

```diff
@@ -118,8 +118,10 @@ public void createOrUpdateDraftApplication_updatesExistingDraft() {
     // Since this is a draft application, it shouldn't have a submit time set.
     assertThat(appDraft2.getSubmitTime()).isNull();
     // Applicant data is not saved in the application until it is submitted.
-    assertThat(appDraft1.getApplicantData().getApplicantName()).isEmpty();
-    assertThat(appDraft2.getApplicantData().getApplicantName()).isEmpty();
+    // Though we are not saving the applicant name to a well known path anymore,
```

Comment:
suggestion (non-blocking): delete the extra comment. It's helpful in the context of this PR but will be come outdated as soon as we remove the WKP code paths.